routing/blinding.go

```diff
@@ -197,22 +231,51 @@ func (s *BlindedPaymentPathSet) FinalCLTVDelta() uint16 {
 
 // LargestLastHopPayloadPath returns the BlindedPayment in the set that has the
 // largest last-hop payload. This is to be used for onion size estimation in
-// path finding.
-func (s *BlindedPaymentPathSet) LargestLastHopPayloadPath() *BlindedPayment {
+// path finding. It also returns the cipher text (encrypted data) of the last
+// hop.
+func (s *BlindedPaymentPathSet) LargestLastHopPayloadPath() (*BlindedPayment,
```

Comment:
I'm not sure this change in complexity is worth it just to be able to add another hop during pathfinding